packages/jenga/jenga.v0.9.0/descr

```diff
@@ -0,0 +1 @@
+Build system
```

Comment:
a tiny bit of elaboration on this (and perhaps a redirect in the descr to jbuilder for the curious casual observer?)